delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutesSpec.scala

```diff
@@ -394,6 +395,27 @@ class ResourcesRoutesSpec extends BaseRouteSpec {
       }
     }
 
+    "validate a resource successfully against its latest schema" in {
```

Comment:
We need a test to get the result when we validate with the special `unconstrained` case